conda/conda-osx-64.lock.yml

```diff
@@ -0,0 +1,360 @@
+# Generated by conda-lock.
+# platform: osx-64
+# input_hash: dd4cd8b2bfa13dad2a079f834ddc75bc178b28ecc4cb931ce6669470d2ec2619
```

Comment:
@scottyhq, what are your thoughts on having an OSX-specific lockfile? Do we need a `conda-osx-64.lock.yml` file or could we remove it?

For the hackweeks, it made sense to have OSX (and possibly Windows) since the hackweek participants may want to install a virtual environment locally on their laptop. For the hosted Cryo JupyterHub though, it will just be using the Linux-built docker image, and I'm not sure if there's the expectation that users will be copying this environment locally?

Reply:
Decided to remove the osx lockfile for now (see c274c75256842f11878e3e41eb0d14f06bc053c0) to keep things minimal. Can always add it back later if requested.